From da966d8d956bb9a47b44b63e6ceaf589d2fdbdf5 Mon Sep 17 00:00:00 2001
From: Harry Youd <harry@harryyoud.co.uk>
Date: Sat, 3 Jun 2017 19:33:51 +0100
Subject: [PATCH 178/214] Remove # from post titles

Some news aggregators don't handle HTML entities in titles
Let's fix that because it looks silly otherwise

Change-Id: I9c091e648923d649248e2630e83421967b7665c0
---
 _posts/2017-05-02-Changelog-8.md  | 2 +-
 _posts/2017-05-15-Changelog-9.md  | 2 +-
 _posts/2017-05-31-Changelog-10.md | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/_posts/2017-05-02-Changelog-8.md b/_posts/2017-05-02-Changelog-8.md
index c828de6..d017c18 100644
--- a/_posts/2017-05-02-Changelog-8.md
+++ b/_posts/2017-05-02-Changelog-8.md
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Changelog &num;8 - May ASB, Jelly, Donations and More Servers
+title: Changelog 8 - May ASB, Jelly, Donations and More Servers
 category: blog
 excerpt: Changes since 18th April 2017
 author: harryyoud
diff --git a/_posts/2017-05-15-Changelog-9.md b/_posts/2017-05-15-Changelog-9.md
index 8a5e6f5..d14f5b7 100644
--- a/_posts/2017-05-15-Changelog-9.md
+++ b/_posts/2017-05-15-Changelog-9.md
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Changelog &num;9 - Gello, Jelly and a security advisory
+title: Changelog 9 - Gello, Jelly and a security advisory
 category: blog
 excerpt: Changes since 2nd May 2017
 author: harryyoud
diff --git a/_posts/2017-05-31-Changelog-10.md b/_posts/2017-05-31-Changelog-10.md
index 7237a23..8fae162 100644
--- a/_posts/2017-05-31-Changelog-10.md
+++ b/_posts/2017-05-31-Changelog-10.md
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Changelog &num;10 - Sensitive numbers and our CVE Tracker
+title: Changelog 10 - Sensitive numbers and our CVE Tracker
 category: blog
 excerpt: Changes since 15th May 2017
 author: harryyoud
-- 
2.17.1

